formats/json/jvmJava9/src/module-info.java

```diff
@@ -0,0 +1,7 @@
+module kotlinx.serialization.json {
+    requires transitive kotlin.stdlib;
+    requires transitive kotlinx.serialization.core;
```

Comment:
Although json definitely uses some things from `kotlinx.serialization.internal` package, it isn't mentioned in 'requires' here. Can this cause issues?

If not, I'd prefer removing `exports kotlinx.serialization.*.internal` compiletely.

Reply:
This line requires the `kotlinx.serialization.core` module, not the package. Since the `kotlinx.serialization.core` module exports the `kotlinx.serialization.internal` package it can be used by `kotlinx.serialization.json`. 

As an alternative, we could export `kotlinx.serialization.internal` to only the format modules.